### DISCUSSION TITLE
Climate Coach：开源团队氛围与包容性社区健康状况监控仪表板

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#298] [3️⃣ 聊学术]
早上在 GitHub 上看到 [Sean P. Goggins](https://github.com/sgoggins) 教授 forked 了一个 [climate_coach](https://github.com/sgoggins/climate_coach) 的项目，是 CMU [STRUDEL ](https://cmustrudel.github.io/)实验室的一个最新工作。

[这篇论文](https://dl.acm.org/doi/10.1145/3544548.3581317)的摘要：开源软件项目已经成为我们日常生活中不可或缺的一部分，几乎支持我们今天使用的所有软件。由于开源软件构成了数字基础设施，因此维护它们至关重要。我们推出了 Climate Coach，这是一个仪表板，可以帮助开源项目维护者在团队氛围和包容性方面监控社区的健康状况。通过文献综述和探索性调查 (N=18)，我们确定了可以反映项目健康状况的重要信号，并将其显示在仪表板上。我们通过两轮有声思考研究评估并完善了我们的仪表板 (N=19)。然后，我们进行了为期两周的纵向日记研究 (N=10)，以测试我们的仪表板的实用性。

仪表板链接：https://www.sophiehsqq.com/climate_coach/index_id.html

<div align=center>
<img src="https://github.com/X-lab2017/open-wonderland/assets/15010826/fea16443-281d-4fa4-b64b-18b286f8b935" width="600px">
</div>

研究对象、研究问题、研究方法、研究成果等均值得参考~

另外，[第一作者](https://www.sophiehsqq.com/index.html)去年完成的[博士论文](https://www.sophiehsqq.com/papers/huilianq_phd_ISR_2022.pdf)，也是个不错的参考~

**Comment by xiaoya-yaya** [comment]
很有借鉴意义
